src/test/java/com/epam/tc/hw2/ex1/ExerciseFirstTest.java

```diff
@@ -0,0 +1,75 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BrowserTestBase;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+
+public class ExerciseFirstTest extends BrowserTestBase {
+    // 5. Assert that there are 4 items on the header section are displayed and they have proper texts
+    @Test
+    public void test4ItemsHeader() {
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(driver.findElement(By.partialLinkText("HOME")).getText())
```

Comment:
Please, use as() method for EVERY assertion in order to provide some description.